docs/source/developers/python.rst

```diff
@@ -302,10 +302,24 @@ created above (stored in ``$ARROW_HOME``):
 
 .. code-block::
 
-   $ mkdir arrow/cpp/build
```

Comment:
The mkdir needs to be kept? Or does cmake create that if not yet existing with the `-B` argument?

Reply:
CMake creates it